maze/README.rst

```diff
@@ -0,0 +1,78 @@
+Maze Solver Demo
+================
+Getting the D-Wave Quantum Computer to solve a maze!
+
+The purpose of this demo is to take a simple and familiar problem - solving a maze - and go through
+the steps of submitting the problem to the quantum computer.
+
+Code Overview
+-------------
+The idea is to describe the rules of how one can move through a maze with a set of constraints.
+Using the Ocean toolkit, these constraints can be converted into a binary quadratic model (BQM)
+that can then be solved with a D-Wave quantum computer. The solution that gets returned by the
+quantum computer is the path needed to get through the maze.
+
+There are several constraints involved with a maze:
+ - Specify valid path moves (ie if the path enters a grid point, it must also leave said grid point)
+ - Path has a specific start and end position
```

Comment:
We want parallel bullets. so something like:
* Valid movements (i.e., if ...
* Paths cannot pass borders 
* Paths ... internal walls